projects/hslayers/src/assets/locales/en.json

```diff
@@ -851,6 +851,7 @@
     "dndLayers": "Use drag&drop to move layers between the map sides!",
     "verticalSwipe": "Vertical swipe",
     "disableSwipe": "Disable swipe",
-    "enableSwipe": "Enable swipe"
+    "enableSwipe": "Enable swipe",
+    "fullMap": "Full map"
```

Comment:
"Full map" does not feel as the best keyword for this. I would go with "Entire map". Or "Whole map", "Complete map", "Full-screen map"...

Reply:
Thanks for clearing that for me, I was struggling with that as well.